src/status_im/ui/screens/profile/user/sheet/views.cljs

```diff
@@ -0,0 +1,49 @@
+(ns status-im.ui.screens.profile.user.sheet.views
+  (:require [re-frame.core :as re-frame]
+            [status-im.i18n :as i18n]
+            [status-im.ui.components.action-button.action-button :as action-button]
+            [status-im.ui.components.action-button.styles :as action-button.styles]
+            [status-im.ui.components.colors :as colors]
+            [status-im.ui.components.react :as react]
+            [status-im.utils.utils :as utils]))
+
+(defn hide-sheet-and-dispatch [event]
+  (re-frame/dispatch [:bottom-sheet/hide-sheet])
+  (re-frame/dispatch event))
+
+(defn profile-icon-actions [include-remove-action?]
+  (fn []
```

Comment:
why do you return fn here? it should be `fn [_]` if you intentionally don't want to use `include-remove-action?` when it's updated

Reply:
learn't it from
https://github.com/status-im/status-react/blob/94373993cfd516b80f48599fbf30aab1931c8e57/src/status_im/ui/screens/wallet/accounts/sheets.cljs#L12-L14
somehow parent component expects 2nd order reagent component
will do the `underscore` thing in function args 👍

Reply:
oh ok , you don't need to change anything then, that's fine, I thought it's a regular view

Reply:
👍